### ISSUE TITLE
Is it possible to work with "assets" folder where files are NOT hashed, but just passed "as is"?

### ISSUE THREAD

**Issue by kirillgroshkov** [parcel-bundler/parcel#1980]
My use case is that I want to have folder `/assets/fonts` with static font files that NEVER change, to cache them for 1 year with http headers, and to NOT have their filenames change with hashes (e.g `LabGrotesqueWeb-BoldItalic.9e1edde9.ttf`). For example, I want to host this file with it's original filename `LabGrotesqueWeb-BoldItalic.ttf`, to reference it from other projects by it's absolute url where it's hosted.

In other words - selectively disable hashing for certain types of resources (e.g for `*.ttf`)

**Comment by kirillgroshkov** [comment]
Ok, clear enough.

**Comment by elbotho** [comment]
@kirillgroshkov  you might want to check out https://github.com/elwin013/parcel-plugin-static-files-copy
